etc/config/fortran.amazon.properties

```diff
@@ -729,6 +729,18 @@ compiler.flangtrunk-fc.supportsBinaryObject=false
 compiler.flangtrunk-fc.supportsBinaryExecute=false
 compiler.flangtrunk-fc.alias=flangtrunknew-fc
 
+#################################
+# LFortran
+group.lfortran.compilers=lfortran
+group.lfortran.groupName=LFortran
+
+compiler.lfortran.name=lfortran (latest)
+compiler.lfortran.exe=/opt/compiler-explorer/lfortran/bin/lfortran
+compiler.lfortran.options=--generate-object-code
+compiler.lfortran.supportsBinary=true
+compiler.lfortran.supportsExecute=true
+compiler.lfortran.emitDebugInfo=false
```

Comment:
What does this mean? Why would you not want debug information for this fortran?

Does it not support the `-g` flag? in which case maybe calling this `supportsDebugInfo` sounds more appropriate? If it's something else, let me know.

Reply:
Thinking about this: we can add a new "compiler type" of lfortran instead, which can then override and _not_ add the `-g`

Reply:
Done